src/main/java/mops/businesslogic/GroupServiceProdImpl.java

```diff
@@ -0,0 +1,67 @@
+package mops.businesslogic;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import mops.exception.MopsException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+@Profile("prod")
+public class GroupServiceProdImpl implements GroupService {
+    /**
+     * URL to GruppenFindung.
+     */
+    @Value("${material1.mops.gruppenfindung.url}")
+    private static String gruppenFindungUrl;
+
+    /**
+     * Directory Service.
+     */
+    private DirectoryService directoryService;
+
+    /**
+     * Allows to send REST API calls.
+     */
+    private RestTemplate restTemplate;
+
+    /**
+     * Getter for url to GruppenFindung.
+     *
+     * @return url string
+     **/
+    public static String getGruppenFindungUrl() {
+        return gruppenFindungUrl;
+    }
```

Comment:
why do you need this getter?

Reply:
because auf checkstyle or pmd can remember

Reply:
if pmd tells you jump down this cliff, will you do it?
just ignore that :P

Reply:
and remove the static

Reply:
I need the static one for the tests.

Reply:
no